.circleci/config.yml

```diff
@@ -1,140 +1,28 @@
-                    govcms/govcms:1.x \
-                    govcms/scaffold-tooling:dev-"${branch}" \
-                    govcms/require-dev:dev-"${branch}" \
-                    symfony/event-dispatcher:"v4.3.11 as v3.4.35" # @todo: remove once govcms/govcms no longer requires "symfony/event-dispatcher:v4.3.11 as v3.4.35" which only works at the root composer.json level.
-              else
-                echo -e "--> Expected stable versions, based on the satis config: \n     - govcms/govcms:${VERSION_GOVCMS} \n     - govcms/scaffold-tooling:${VERSION_TOOLING} \n     - govcms/require-dev:${VERSION_DEV}"
-                composer require --no-update \
-                    govcms/govcms:"${VERSION_GOVCMS}" \
-                    govcms/scaffold-tooling:"${VERSION_TOOLING}" \
-                    govcms/require-dev:"${VERSION_DEV}" \
-                    symfony/event-dispatcher:"v4.3.11 as v3.4.35" # @todo: remove once govcms/govcms no longer requires "symfony/event-dispatcher:v4.3.11 as v3.4.35" which only works at the root composer.json level.
-              fi
-              cat composer.json | jq .require
-              composer -n --quiet --no-suggest --no-scripts update
-              composer info | grep ^govcms
-            done
-
-  deploy:
-    docker:
-      - image: quay.io/govcms/govcms-ci
+  test_develop:
+    <<: *job-test
     environment:
-      SATIS_BUILD: /tmp/satis-build/app
-    steps:
-      - checkout
-      - attach_workspace:
-          at: /tmp/satis-build
-      - run:
-          name: Update github develop branch.
-          command: |
-            # If builds were automated, this would push the new satis back to git.
-            exit 1
-
-            # Currently just testing a push to a test branch.
-            git checkout -b test-"${CIRCLE_SHA1}"
-            rm -Rf app
-            cp -Rf "${SATIS_BUILD}" .
-            git add app
-            git config --global user.email "govhosting@finance.gov.au"
-            git config --global user.name "GovCMS service account"
-            # Insecure proof of concept. Key removed between testing.
-            # @todo use ssh key fingerprint https://github.com/integratedexperts/drupal-dev/blob/8.x/.circleci/config.yml#L127
-            git remote set-url origin https://simesy:"${GITHUB_READ_WRITE_TOKEN}"@github.com/govCMS/satis.git
-            git commit -m"[skip ci] test-commit"
-            git push origin test-"${CIRCLE_SHA1}"
+      SATIS_BRANCH: develop
 
 
 workflows:
   version: 2.1
   update:
     jobs:
-      - build:
-          filters:
-            tags:
-              ignore: /.*/
-      - test:
-          filters:
-            tags:
-              ignore: /.*/
-          requires:
-            - build
-      - deploy:
-          filters:
-            tags:
-              ignore: /.*/
-            branches:
-              only:
-                - upstream_changes
-                - develop
-          requires:
-            - test
+      - test
```

Comment:
3 parallel jobs allows to reduce build time to ~2 mins and make sure that there are no artifacts from the previous build (clean run)